### DISCUSSION TITLE
Access to /dev/fuse from inside Jail

### DISCUSSION THREAD

**Discussion by PandaWinn** [Jip-Hop/jailmaker#213] [General]
Thanks again, Capt'n and everyone involved to make this solution happen. This has been a game changer for me.

I'm running an instance of Nextcloud-aio (NC-AIO) in a docker jail, using the standard script. GPU passthrough works, has been stable and host mounts have been problem-free.

NC-AIO has a built-in backup solution using BorgBackup to create incremental backups. I want to get this backing up to another pool (different disks) on my Truenas machine. However, according to the documentation, BorgBackup needs access to fuse on the host (/dev/fuse).

I get the following error when trying to run it: 
`Message: Could not start container nextcloud-aio-borgbackup: Server error: "POST http://127.0.0.1/v1.41/containers/nextcloud-aio-borgbackup/start" resulted in a "500 Internal Server Error" response:
{&quot;message&quot;:&quot;error gathering device information while adding custom device \&quot;/dev/fuse\&quot;: no such file or directory&quot;}
File: /var/www/docker-aio/php/src/Docker/DockerActionManager.php`

Is there a way to access /dev/fuse from inside the Jail?
Is it as simple as mounting it as a bind?

Sorry if this is simple, this has been an amazing opportunity to learn my way around linux and I'm still getting started.

**Comment by Jip-Hop** [comment]
Yes it is as simple as that!
